tools/schemacode/setup.cfg

```diff
@@ -58,6 +58,7 @@ bidsschematools =
     data/schema/SCHEMA_VERSION
     data/schema/**/*.yaml
     tests/data/**/*
+    tests/data/**/.bidsignore
```

Comment:
* There are currently no directories under [tools/schemacode/bidsschematools/tests/data](https://github.com/bids-standard/bids-specification/tree/master/tools/schemacode/bidsschematools/tests/data); where is this test data coming from?  CC @TheChymera
* If you just want to include everything under `bidsschematools` in the sdist & wheel, I would recommend forgoing the fiddly `package_data` specs and instead setting `include_package_data = True` alongside [a `MANIFEST.in` file](https://packaging.python.org/en/latest/guides/using-manifest-in/) with contents like:

        graft bidsschematools
        prune bidsschematools/tests/data/**/.git  # Assuming git repos are actually involved
        global-exclude *.py[cod]

Reply:
So we have fixtures to pull down the current BIDS examples OR serve them from package data: https://github.com/bids-standard/bids-specification/blob/master/tools/schemacode/bidsschematools/conftest.py

@TheChymera wanted to make sure that when running the tests from downloaded packages, tests would not fail because the examples changed. So there's a special test that populates the package data: https://github.com/bids-standard/bids-specification/blob/master/tools/schemacode/bidsschematools/tests/test_make_testdata.py That gets run during release builds.

Reply:
I think @effigies explained what we're doing here quite well and succinctly :)

As for the suggestions:

@yarikoptic `.*` was my first instinct as well, but no. We don't want `.git`, and we explicitly exclude it [here](https://github.com/bids-standard/bids-specification/blob/8c2cb849e2a18d8c37838158b055a8173221bbfa/tools/schemacode/bidsschematools/tests/test_make_testdata.py#L37) as well.

`bids-examples` and `bids-error-examples` are populated with data stubs, so there is no point in using datalad here.